doc/user_documentation.rst

```diff
@@ -293,6 +295,24 @@ of the names ``<coprname>:pr:<pr_id>``. ``<pr_id>`` is ID of the pull request op
 
     $ sudo dnf copr enable <ownername>/<coprname>:pr:<pr_id>
 
+Create build from PR comment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+How to set up your project in Pagure:
+
+- Go to ``Settings`` on your project
+- Go to ``Project Options`` section
+- Select ``Fedmsg notifications`` option
+
+How to se up project in Copr:
+
+- Create new package in your Copr repository
+- Select ``SCM`` source type and specify ``Clone url``
+- Check the ``Auto-rebuild`` section
```

Comment:
The`[copr-build]` keyword doesn't need a specific project configuration actually.  It is automatically available with the `Auto-rebuilding` steps above.  This seems redundant, or could be at least "merged" into the steps above?

Reply:
I did some research and the `auto-rebuilding` part and `PR-comment` rebuilding part really differ.  If you disable the `Fedmsg` in `Hooks` then `Auto-rebuilding` stop working. And if you disable ` Fedmsg notifications ` in `Project option` then PR-comment rebuild stops working. These really are two independent settings.

Also from `Fedmsg` option in `Hooks`:
```
This hook pushes commit notification to the fedmsg bus to be consumed by other
applications.

It is different from the fedmsg setting present in the project options block
which publishes notifications about events happening in the project via
pagure's (web) user interface, for example: new tickets, new comments, new
pull-requests and so on.
This hook on the other hand only acts on commits.
```

So what is says is the one option is really for Pagure-UI related things (things you do in browser) and the other is for commits etc.